docs/_docs/01-ug-quick-start-guide.md

```diff
@@ -240,15 +240,52 @@ hs.explain(query, verbose = True)
 #### Enable Hyperspace
 
 Now that you have created an index that your query can utilize, you can enable Hyperspace and execute your query:
-
-Scala:
-```scala
-spark.enableHyperspace
-query.show
-```
-
-Python:
-```python
-Hyperspace.enable(spark)
-query.show()
-```
+There are two ways of enabling Hyperspace.
+
+1. Using Hyperspace extension
```

Comment:
Can we enable Hyperspace by specifying the extension in `spark-defaults.conf`? 
If so, I think it's worth to mention it.

Reply:
Where can I test it? Can I test it with Synapse?

Reply:
Can you try https://techcommunity.microsoft.com/t5/azure-synapse-analytics/how-to-set-spark-pyspark-custom-configs-in-synapse-workspace/ba-p/2114434?

Reply:
I tried to set up cluster with config.txt that contains `spark.sql.extensions com.microsoft.hyperspace.HyperspaceSparkSessionExtension` and set the jar with HyperspaceExtension.

I built indexes before executing example code and I succeed to use indexes with code without explicitly import Hyperspace.
So, I will add more details regarding `spark-defaults.conf` too.